checker/src/org/checkerframework/checker/minlen/MinLenTransfer.java

```diff
@@ -0,0 +1,584 @@
+                        }
+                    }
+                }
+            }
+        }
+
+        return result;
+    }
+
+    @Override
+    public TransferResult<CFValue, MinLenStore> visitMethodInvocation(
+            MethodInvocationNode node, TransferInput<CFValue, MinLenStore> in) {
+        ProcessingEnvironment env = atypeFactory.getProcessingEnv();
+        TransferResult<CFValue, MinLenStore> result = super.visitMethodInvocation(node, in);
+
+        String methodName = node.getTarget().getMethod().toString();
+        boolean add = methodName.startsWith("add(");
+        boolean asList = methodName.contains("asList(");
+        boolean toArray = methodName.startsWith("toArray(");
+        if (!(add || asList || toArray)) {
+            return result;
+        }
+
+        if (TreeUtils.isMethodInvocation(node.getTree(), listAdd, env)
+                || TreeUtils.isMethodInvocation(node.getTree(), listAdd2, env)) {
+            Receiver rec =
+                    FlowExpressions.internalReprOf(
+                            analysis.getTypeFactory(), node.getTarget().getReceiver());
+            if (node.getTarget().getReceiver().getTree() == null || rec instanceof Unknown) {
+                return result;
+            }
+            AnnotatedTypeMirror ATM =
+                    atypeFactory.getAnnotatedType(node.getTarget().getReceiver().getTree());
+            AnnotationMirror anno = ATM.getAnnotation(MinLen.class);
+            if (anno == null || AnnotationUtils.areSameByClass(anno, MinLenBottom.class)) {
+                return result;
+            }
+            int value = MinLenAnnotatedTypeFactory.getMinLenValue(anno);
+            AnnotationMirror AM = atypeFactory.createMinLen(value + 1);
+            Set<AnnotationMirror> set = new HashSet<>();
+            set.add(AM);
+            CFValue minlen = new CFValue(analysis, set, node.getTarget().getReceiver().getType());
+            if (MinLenStore.canInsertReceiver(rec)) {
+                if (result.containsTwoStores()) {
+                    result.getThenStore().replaceValue(rec, minlen);
+                    result.getElseStore().replaceValue(rec, minlen);
+                } else {
+                    result.getRegularStore().replaceValue(rec, minlen);
+                }
+            }
+            return result;
+        } else if (TreeUtils.isMethodInvocation(node.getTree(), listToArray, env)
+                || TreeUtils.isMethodInvocation(node.getTree(), listToArray1, env)) {
+            if (node.getTarget().getReceiver().getTree() == null) {
+                return result;
+            }
+            AnnotatedTypeMirror ATM =
+                    atypeFactory.getAnnotatedType(node.getTarget().getReceiver().getTree());
+            AnnotationMirror anno = ATM.getAnnotation(MinLen.class);
+            int value = MinLenAnnotatedTypeFactory.getMinLenValue(anno);
+            AnnotationMirror AM = atypeFactory.createMinLen(value + 1);
```

Comment:
Because this transfer is for >. If something is greater than a @MinLen(3), it should be @MinLen(4), for instance.